Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:10-alpine as deps
```

Comment:
Can we use LTS node here

Reply:
10 is LTS, the package.json is specifying v10 as well, can upgrade if needed

Reply:
My only thought was the using `node:lts` saves us one more line of config to change in the future

Reply:
Correct, but lts version changes over time, unless we remove the 
`"engines": {
    "node": "10"
  },`
from the package file then builds will fail once lts changes in the future, either remove the requirements from the package file or keep them consistent in both places

Reply:
Yep you're right that makes a lot of sense